battleship.py

```diff
@@ -89,9 +109,16 @@ def checkShip(grid, ship):
 Returns: 2D list of ints
 '''
 def addShips(grid, numShips):
-    return
-
+    count=0
+    while count<numShips:
+        newship = createShip()
+        if checkShip(grid,newship):
+          for j in newship:
+                  grid[j[0]][j[1]]=2

```

Comment:
Use variable name instead of the number.

Reply:
issue resolved